src/pages/AppConfigure.vue

```diff
@@ -537,6 +538,7 @@ export default {
     let password = ref("");
     let passwordMessage = ref("");
     let passwordMessageType = ref("");
+    let userLimitError = false;
```

Comment:
Since this value will change, should we make it a `ref`?

Reply:
We're not using this anywhere in template, hence I thought not add `ref` to it

Reply:
Okay 👌🏼